src/main/java/com/microsoft/azure/spring/data/cosmosdb/common/PropertyLoader.java

```diff
@@ -34,7 +34,7 @@ public static boolean isApplicationTelemetryAllowed() {
         if (allowed == null) {
             return true;
         } else {
-            return allowed.equalsIgnoreCase("false") ? false : true;
+            return !allowed.equalsIgnoreCase("false");
```

Comment:
Why not  use `allowed.equalsIgnoreCase("true")`?

Reply:
because if user input some mess like True/TRUE will be treated as false but "false" will be treated as true.